pkg/autodetect/main.go

```diff
@@ -79,6 +81,11 @@ func (b *Background) autoDetectCapabilities() {
 	}
 
 	b.detectClusterRoles()
+	if err := b.cleanDeployments(); err != nil {
+		log.WithFields(log.Fields{
+			"error": err,
+		}).Errorf("Error cleaning orphaned deployment")
```

Comment:
`s/Error/error/`
`s/deployment/deployments/`, but I think the method should not fail, just like the other `detect` methods. Instead, log the specific messages within the `cleanDeployments` function.